src/openrct2/world/TileInspector.cpp

```diff
@@ -151,12 +151,15 @@ GameActionResult::Ptr tile_inspector_remove_element_at(const CoordsXY& loc, int1
 {
     if (isExecuting)
     {
-        // Forcefully remove the element
-        TileElement* const tileElement = map_get_nth_element_at(loc, elementIndex);
-        if (tileElement == nullptr)
+        // At least one tile element must exist on a tile
+        if (windowTileInspectorElementCount <= 1)
```

Comment:
Although this does work this is not network safe. This variable is not passed over the network. You will need to recalculate the number of elements in this function to make it safe. Also this check only needs to be performed when in the non isExecuting branch.

Reply:
@Broxzier thanks for the lead. I plan on working this, this weekend. Will let you know if there's any questions